CMakeLists.txt

```diff
@@ -286,6 +287,17 @@ endif(NOT HAVE_LIBVNCSERVER_IN_ADDR_T)
 
 TEST_BIG_ENDIAN(LIBVNCSERVER_WORDS_BIGENDIAN)
 
+if(WITH_SASL)
+  find_package(PkgConfig)
```

Comment:
pls find a way to use something else than pkgconfig. i now we use it to find systemd, but that only exists on linux where pkgconfig mostly already is installed. libsasl, otoh, might get used on windows and osx as well